ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CLIENT_ID</key>
+	<string>123456789012-12345678901234567890123456789012.apps.googleusercontent.com</string>
+	<key>REVERSED_CLIENT_ID</key>
+	<string>com.googleusercontent.apps.123456789012-12345678901234567890123456789012</string>
+	<key>API_KEY</key>
+	<string>AasdfDEzghasdfghaEsdfEFghRasdsfghasdfgh</string>
+	<key>GCM_SENDER_ID</key>
+	<string>123456789012</string>
```

Comment:
```suggestion
	<key>CLIENT_ID</key>
	<string>Your client id here</string>
	<key>REVERSED_CLIENT_ID</key>
	<string>Your reversed client id here</string>
	<key>API_KEY</key>
	<string>your API key here</string>
	<key>GCM_SENDER_ID</key>
	<string>your sender id here</string>
```

Reply:
removed creds, they should be substituted by github actions

Reply:
please add that file to `.gitignore` so you would be free to set any creds locally without commiting them to the repo

Reply:
All these were fake, just so that it passes the checks done by the module, otherwise it would fail the builds.
With the new values it will fail those checks too, because of their format